src/handle_wrap.cc

```diff
@@ -69,6 +81,7 @@ HandleWrap::HandleWrap(Environment* env,
   HandleScope scope(env->isolate());
   Wrap(object, this);
   env->handle_wrap_queue()->PushBack(this);
+  env->SetMethod(object, "isRefed", HandleWrap::IsRefed);
```

Comment:
This is not the way it's done. This has to be declared in every child class: https://github.com/nodejs/node/blob/v5.9.0/src/tcp_wrap.cc#L87

Otherwise you're setting an object property on an already constructed object.